### ISSUE TITLE
Cuesheets in Playlist

### ISSUE THREAD

**Issue by preterive** [savonet/liquidsoap#852]
**Is your feature request related to a problem? Please describe.**
I play dj mixes on my Station. I have cuesheet files for every mix, but i can't send the Metadata.
**Describe the solution you'd like**
I would really like a way to create a Playlist file (for example m3u) with all cue sheet files.

**Comment by smimram** [comment]
What do you mean exactly by cue sheet? Like the cue in / out points? If so you could have a playlist with lines of the form
```
annotate:liq_cue_in=10.,liq_cue_out=20.:/my/mp3/file.mp3
```

**Comment by preterive** [comment]
no, i mean a feature to add many cue Sheet file paths to a m3u playlist.

**Comment by smimram** [comment]
I don't understand what you mean exactly by "cue sheet file".

**Comment by preterive** [comment]
I mean a way to add .cue files in a m3u Playlist.

**Comment by davidtrattnig** [comment]
@preterive @smimram I have a similar need to include Cue Sheets inside M3U playlists.

Alternatively (or maybe even more elegant solution) it could be done with multiple `FILE` block in Cue Sheets: https://github.com/savonet/liquidsoap/issues/1373
